deploy/crds/wildfly_v1alpha1_wildflyserver_crd.yaml

```diff
@@ -87,11 +87,24 @@ spec:
                     type: string
                   podIP:
                     type: string
+                  state:
+                    description: Represent the state of the Pod, it's used especially
+                      during scale down the expected values are represented by the
+                      PodState* constants  Read-only.
+                    type: string
                 required:
                 - name
                 - podIP
+                - state
                 type: object
               type: array
+            scalingdownPods:
```

Comment:
please update also documentation in doc dir?

Reply:
I added text there. Thanks for the hint.